src/App.jsx

```diff
@@ -14,33 +18,92 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+const SORT_ALPHABETICALLY = 'Sort alphabetically';
+const SORT_BY_LENGTH = 'Sort by length';
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+function getPreparedGoods(goods, { sortField, reversed }) {
+  let preparedGoods = [...goods];
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+  if (sortField) {
+    preparedGoods.sort((good1, good2) => {
+      if (sortField === SORT_ALPHABETICALLY) {
+        return good1.localeCompare(good2);
+      }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+      if (sortField === SORT_BY_LENGTH) {
+        return good1.length - good2.length;
+      }
+
+      return 0;
+    });
+  }
+
+  if (reversed) {
+    preparedGoods = preparedGoods.reverse();
+  }
+
+  return preparedGoods;
+}
```

Comment:
The `getPreparedGoods` function currently does not handle cases where an invalid `sortField` is provided. It defaults to returning 0, which means no sorting will be applied. Consider adding a check or a default case to handle unexpected `sortField` values more explicitly, possibly by logging a warning or throwing an error.